x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_creation_ui/pages/rule_editing/index.tsx

```diff
@@ -509,6 +511,16 @@ const EditRulePageComponent: FC<{ rule: RuleResponse }> = ({ rule }) => {
     [navigateToApp, ruleId]
   );
 
+  const upgradeCallout = useRuleUpdateCallout({
```

Comment:
We mostly stick to `upgrade` word in the codebase while using `update` in texts exposed to users. I recommend to stick to `upgrade` in names you used for consistency.